pkg/networkgraph/tree/radix.go

```diff
@@ -410,6 +410,97 @@ func (t *nRadixTree) findCIDRNoLock(ipNet *net.IPNet) (*nRadixNode, error) {
 	return ret, nil
 }
 
+func validateLeavesHaveValues(node *nRadixNode) bool {
```

Comment:
All of this "validate" stuff is just used for testing. I confirmed that the added unit tests fail when the old code is used.